src/renderer/components/dock/dock-tab.tsx

```diff
@@ -88,13 +88,13 @@ export class DockTab extends React.Component<DockTabProps> {
     const { className, moreActions, ...tabProps } = this.props;
     const { title, pinned } = tabProps.value;
     const label = (
-      <div className="flex gaps align-center">
+      <div className="flex gaps align-center" onAuxClick={isMiddleClick(prevDefault(this.close))}>
         <span className="title" title={title}>{title}</span>
         {moreActions}
         {!pinned && (
           <Icon
             small material="close"
-            title="Close (Ctrl+W)"
+            title="Close (Ctrl+Shift+W)"
```

Comment:
Is this what is currently used anyway?

Reply:
Yeah. The Shift Key was indicated by the uppercase "W" instead of a lowercase one I suppose. On Windows and Linux I wasn't able to close the tab without pressing the shift key

Reply:
Oh I figure that any keys in such hints are case insensitive anyway. Thanks for this change.